python/basix/ufl_wrapper.py

```diff
@@ -351,6 +360,17 @@ def is_custom_element(self) -> bool:
         """True if the element is a custom Basix element."""
         return self._is_custom
 
+    @property
+    def has_tensor_product_factorisation(self) -> bool:
+        """True if the element has a tensor product factorisation."""
+        return self.element.has_tensor_product_factorisation
+
```

Comment:
It’s not clear from the docstring what is returned. Could some more explanation be added?

Reply:
@mscroggs?

Reply:
I've added a bit of detail to the docstring